pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -69,8 +78,21 @@ type CronWorkflowStatus struct {
 	LastScheduledTime *metav1.Time `json:"lastScheduledTime" protobuf:"bytes,2,opt,name=lastScheduledTime"`
 	// Conditions is a list of conditions the CronWorkflow may have
 	Conditions Conditions `json:"conditions" protobuf:"bytes,3,rep,name=conditions"`
+	// Succeeded is a counter of how many times the child workflows had success
+	Succeeded int64 `json:"succeeded" protobuf:"varint,4,rep,name=succeeded"`
+	// Failed is a counter of how many times a child workflow terminated in failed or errored state
```

Comment:
```suggestion
	// v3.6 and after: Failed counts how many times a child workflow failed
```
- version specifier
- a bit simpler
- ~"terminated" to "completed" -- "Terminate" is an action on Workflows that has a specific meaning. "Completed" has the correct meaning here~.
  - we can just say "failed", which describes this and is similarly an existing term